jenkins-design-language/package.json

```diff
@@ -41,6 +41,7 @@
     "url": "https://github.com/jenkinsci/jenkins-design-language.git"
   },
   "dependencies": {
+    "linkifyjs": "2.1.4",
```

Comment:
@sophistifunk this one? https://github.com/SoapBox/linkifyjs/blob/master/package.json so doesn't bring anything along, which is nice, right? (ie fairly small)

Reply:
Yeah there's not a lot of code in there. It has a dependency on JQ, but only for functions we're not using so we can just ignore it.

Reply:
I eyeballed the deps too and looks fine. I think as long as we verify that the bundles aren't bloating up after this change we are good. They do strange things sometimes :/